floodlight/io/dfl.py

```diff
@@ -513,24 +563,48 @@ def read_event_data_xml(
             f"other segment!"
         )
 
+    # link team1 and team2 to home and away
+    home_tID = home_teamsheet.teamsheet.at[0, "tID"]
+    away_tID = away_teamsheet.teamsheet.at[0, "tID"]
+    links_team_to_role = {
+        "Home": home_tID,
+        "Away": away_tID,
+    }
+
+    # check if home and away tIDs occur in event data
+    if team1 != home_tID and team2 != home_tID:
+        raise AttributeError(
+            f"The tIDs of teams in the event data ({team1} and {team2}) "
+            f"does not match the tID for the home team in the "
+            f"match information {home_tID}!"
```

Comment:
```suggestion
            f"Neither tID of teams in the event data ({team1} and {team2}) "
            f"matches the tID of the home team from the "
            f"match information {home_tID}!"
```

Reply:
Changed "from the match information" to "from the teamsheet" since this error can also occur when passing a teamsheet with the wrong tID

Reply:
Yup!